### PR TITLE
Add new schedule layout

### PR DESCRIPTION
**Pull request by patrick91**
TODO:

- [ ] Mobile list view
- [x] Header to select the day
- [x] Filter by day
- [ ] Restore ICS views
- [ ] My Schedule view
- [ ] Restore list view
- [ ] Voting/Booking

This is based on what we did for PyCon Italia :)

### CODE REVIEW

File: p3/views/schedule/grid.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
+import typing
+from dataclasses import dataclass
+
+
+@dataclass
```
Comment (umgelurgel) [inline p3/views/schedule/grid.py]: Staging and prod are running python 3.6, is there a way to use dataclasses in that version of python?

File: p3/views/schedule/__init__.py [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+    return
+
+
+def schedule(request, conference):
+    # TODO: filter by day
+    sids = Schedule.objects.filter(conference=conference).values_list(
+        "id", flat=True
+    )
+
+    from conference.dataaccess import schedules_data
```
Comment (umgelurgel) [inline p3/views/schedule/__init__.py]: How much work would it be to remove the dependency on `dataaccess`? I think the long term plan was to deprecate that module.

**Review by umgelurgel — COMMENTED** [review]
That's a huge changeset - can't wait to see it in action!

File: p3/views/schedule/grid.py (hunk 1 above)
Comment (patrick91) [reply p3/views/schedule/grid.py]: @umgelurgel sorry I missed this review :) 

dataclasses has a backport package (which I just added to the requirements file) :)

File: p3/views/schedule/__init__.py (hunk 2 above)
Comment (patrick91) [reply p3/views/schedule/__init__.py]: @umgelurgel not really sure, I think I won't have much time for that, the mobile view will take me a bit of time probably, I think we should remove dataccess after ep2019 ;)

**Review by artcz — APPROVED** [review]
Looks Good enough to merge;
Would be nice to move the url to something like `/schedule/` (instead of `/p3/schedule/ep2019/`) and maybe move the functions to `conference/schedule.py`; Second part can be fixed anytime, URL should probably be fixed before going live. ;)

**Comment by umgelurgel** [comment]
I'll update the url and handle moving the files to `conference` before merging.

**Pull request merged**